lib/platform/github/gh-got-wrapper.js

```diff
@@ -5,21 +5,25 @@ let cache = {};
 
 // istanbul ignore next
 function sleep(ms) {
-  // eslint-disable-next-line promise/avoid-new
-  return new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
This perhaps could be more cleanly refactored to use package `delay` like we have in some other files. Then skipping it in test is just a manner of `jest.mock('delay')`